workshops/templates/workshops/all_persons.html

```diff
@@ -17,14 +18,16 @@
 {% if all_persons %}
     <table class="table table-striped">
       <tr>
-        <th width="50">Instructor?</th>
+        <th width="25">DC?</th>
+        <th width="25">SWC?</th>
```

Comment:
I find two columns a lot easier to read. What I'd really like to do is:

1.  Mark particular badges as "instructor-flavored".
2.  Loop over that set to auto-construct the columns for things like this.

but that can wait...